plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -189,6 +189,7 @@ protected QueryRunner createQueryRunner()
         return queryRunner;
     }
 
+    @SuppressWarnings("DuplicateBranchesInSwitch")
```

Comment:
This change is unrelated to "Support for comments on views for Delta Lake". I would recommend separating a commit.